cmd/access/node_builder/staked_access_node_builder.go

```diff
@@ -104,36 +112,60 @@ func (builder *StakedAccessNodeBuilder) enqueueUnstakedNetworkInit(ctx context.C
 
 	builder.Component("unstaked network", func(_ cmd.NodeBuilder, node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
 
-		// NodeID for the staked node on the unstaked network
-		// TODO: set a different node ID of the staked access node on the unstaked network
-		unstakedNodeID := builder.NodeID // currently set the same as the staked NodeID
-
-		// Networking key
-		// TODO: set a different networking key of the staked access node on the unstaked network
-		unstakedNetworkKey := builder.NetworkKey
-
-		libP2PFactory, err := builder.initLibP2PFactory(ctx, unstakedNodeID, unstakedNetworkKey)
+		libP2PFactory, err := builder.initLibP2PFactory(ctx, builder.NodeID, builder.NodeConfig.NetworkKey)
```

Comment:
`unstaked network` component now broken into `middleware` and `network` since the sync engine needs the middleware provider.